src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -22,12 +22,8 @@
 import java.util.Set;
 import java.util.UUID;
 import java.util.stream.Collectors;
-import net.snowflake.ingest.connection.HistoryRangeResponse;
-import net.snowflake.ingest.connection.HistoryResponse;
-import net.snowflake.ingest.connection.IngestResponse;
-import net.snowflake.ingest.connection.IngestResponseException;
-import net.snowflake.ingest.connection.RequestBuilder;
-import net.snowflake.ingest.connection.ServiceResponseHandler;
+
+import net.snowflake.ingest.connection.*;
```

Comment:
nit: Let's not use `*` import.

Reply:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad